test/gtest/cstring.cpp

```diff
@@ -105,4 +105,12 @@ TEST(cstring, substr) {
     ASSERT_TRUE(c.exceptLast(2) == "simple"); // NOLINT
 }
 
+TEST(cstring, replace) {
+    cstring c = "Original";
+    ASSERT_TRUE(c.replace("in", "out") == "Origoutal");  // NOLINT
```

Comment:
Should probably be `EXPECT_EQ`

Reply:
I am not sure what equality operator is called by EXPECT_EQ, that's why I was using this version.
That's why NOLINT is there too. But if you vouch for the right operator then I can change this.

Reply:
I'm pretty sure it is equivalent to writing `==` (it will call the `==` operator for cstring). You can also add a negative test with `EXPECT_NE` to make sure it works as expected :)